options.html

```diff
@@ -30,11 +30,25 @@
 							<br />
 							<input type="checkbox" id="minihelp" name="minihelp">
 							Show mini help
+							<br />
+							<br />
+							Show popup with delay: 
+							<input type="number" id="popupDelay" name="popupDelay" min="1"> milliseconds
 							</p>
 					</div>
 					<!-- <div id="keycon" class="tabcontent hide"> -->
 					<div id="keycon" class="tabContent">
 					<h1>Keyboard</h1>
+						<input type="checkbox" id="showOnKeyFlag" name="showOnKeyFlag" disabled>

```

Comment:
This design seems like it could be simplified.

Instead of a checkbox followed by a select why not use radio buttons where the default value is  "Not used" or similar?